docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+    ```
+    response:
+    ```clojure
+    {"paired?" true, 
+     "has-master-key?" true, 
+     "app-version" "2.2", 
+     "free-pairing-slots" 4, 
+     "pin-retry-counter" 3, 
+     "puk-retry-counter" 5, 
+     "initialized?" true, 
+     "secure-channel-pub-key" "04b079ac513d5e0ebbe9becbae1618503419f5cb59edddc7d7bb09ce0db069a8e6dec1fb40c6b8e5454f7e1fcd0bb4a0b9750256afb4e4390e169109f3ea3ba91d", 
+     "key-uid" "a5424fb033f5cc66dce9cbbe464426b6feff70ca40aa952c56247aaeaf4764a9", 
+     "instance-uid" "2268254e3ed7898839abe0b40e1b4200"}
+    ```
+
+## Tx signing
+1. `status-im.hardwallet.card/get-application-info`
+   params: 
+   ```clojure
+   {:pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+    :on-success :hardwallet/sign}
+   ```
+   response:
+   ```clojure
+    {"paired?" true, 
+     "has-master-key?" true, 
+     "app-version" "2.2", 
+     "free-pairing-slots" 4, 
+     "pin-retry-counter" 3, 
+     "puk-retry-counter" 5, 
+     "initialized?" true, 
+     "secure-channel-pub-key" "0476d11f2ccdad4e7779b95a1ce063d7280cb6c09afe2c0e48ca0c64ab9cf2b3c901d12029d6c266bfbe227c73a802561302b2330ac07a3270fc638ad258fced4a", 
+     "key-uid" "d5c8cde8085e7a3fcf95aafbcbd7b3cfe32f61b85c2a8f662f60e76bdc100718", 
+     "instance-uid" "e20e27bfee115b431e6e81b8e9dcf04c"}
+    ```
+
+2. `status-im.hardwallet.card/sign` 
+   params:
+   ```clojure
+   {:hash "92fc7ef54c3e0c42de256b93fbf2c49dc6948ee089406e204dec943b7a0142a9", 
+    :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+    :pin "123123", 
+    :path "m/44'/60'/0'/0/0"}
+   ```
+   response: `5d2ca075593cf50aa34007a0a1df7df14a369534450fce4a2ae8d023a9d9c0e216b5e5e3f64f81bee91613318d01601573fdb15c11887a3b8371e3291e894de600`
+
+## Account derivation
+1. `status-im.hardwallet.card/verify-pin` 
+    params:
+    ```clojure
+    {:pin "123123", 
+     :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h"}
+    ```
+    response: `3`
+1. `status-im.hardwallet.card/export-key`
+    params:
+    ```clojure
+    {:pin "123123", 
+     :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+     :path "m/44'/60'/0'/0/1"}
+    ```
+    response: `046d1bcd2310a5e0094bc515b0ec995a8cb59e23d564094443af10011b6c00bdde44d160cdd32b4b6341ddd7dc83a4f31fdf60ec2276455649ccd7a22fa4ea01d8` (account's `public-key`)
+    
+## Where are the keys stored?
+1. When we create a regular multiaccount all its keys are stored on device and are encrypted via key which is derived from user's password. In case if account was created using keycard all keys are stored on the card and are retreived from it during signing into multiaccount. 
+2. When we create a regular multiaccount we also create a separate dabase for it and this database is encrypted using key which is derived from user's password. For a keycard account we use `encryption-public-key` (returned by `status-im.hardwallet.card/get-keys`/`status-im.hardwallet.card/generate-and-load-keys`) as a password.
+
```

Comment:
## Copyright

Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).